lib/livebook/live_markdown/export.ex

```diff
@@ -344,4 +344,14 @@ defmodule Livebook.LiveMarkdown.Export do
     keys = [:hub_secret_names]
     put_unless_default(%{}, Map.take(notebook, keys), Map.take(Notebook.new(), keys))
   end
+
+  defp ensure_order(map) do
+    map
+    |> Enum.sort()
+    |> Enum.map(fn
+      {key, %{} = value} -> {key, ensure_order(value)}
+      pair -> pair
+    end)
+    |> Jason.OrderedObject.new()
+  end
```

Comment:
Serialized maps used to have sorted keys, now we sort explicitly, so it's easier to test the exported source.